web/src/main/angular/src/app/core/components/application-list/application-list-data.service.ts

```diff
@@ -15,8 +15,9 @@ export class ApplicationListDataService {
         private http: HttpClient,
     ) {}
 
-    getApplicationList(): Observable<IApplication[]> {
-        return this.http.get<IApplication[]>(this.url).pipe(
+    getApplicationList(force: boolean): Observable<IApplication[]> {
+        const options = force ? {params: {clearCache:'true', from: 'reload', _t: Date.now().toString()}} : {};
+        return this.http.get<IApplication[]>(this.url, options).pipe(
```

Comment:
```suggestion
        const options = force ? {params: {clearCache:'true', from: 'reload', _t: Date.now().toString()}} : {};
        
        return this.http.get<IApplication[]>(this.url, options).pipe(
```
Please add a line-break after declaration.